xcode/Subconscious/Shared/Components/Notebook/Notebook.swift

```diff
@@ -840,8 +840,7 @@ struct NotebookModel: ModelProtocol {
     static func submitSearch(
         state: NotebookModel,
         environment: AppEnvironment,
-        query: String,
-        defaultAudience: Audience = .local
```

Comment:
We did not use this flag, and flagging address type via audience is now deprecated. Removing it.